pages/Types.html

```diff
@@ -657,6 +658,8 @@ <h2 id="XMLHttpRequest"> XMLHttpRequest </h2>
 <h2 id="jqXHR"> jqXHR </h2>
 <p>As of jQuery 1.5, the <a href="/jQuery.ajax/">$.ajax()</a> method returns the jqXHR object, which is a superset of the XMLHTTPRequest object. For more information, see the <a href="/jQuery.ajax/#jqXHR">jqXHR section of the $.ajax entry</a>
 </p>
+<h2 id="Thenable">Thenable</h2>
+<p>Any object that has a <code>then</code> method.</p>
```

Comment:
Hmm, the spec at https://promisesaplus.com/#point-7 mentions functions with a `then` method as acceptable input as well.

That's a little weird one but technically we're still accepting those so maybe we should keep our definition identical to the Promises/A+ one?